_topic_map.yml

```diff
@@ -1155,8 +1155,8 @@ Topics:
     File: persistent-storage-csi-migration
   - Name: AWS Elastic Block Store CSI Driver Operator
     File: persistent-storage-csi-ebs
-  - Name: Azure Disk CSI Driver Operator
-    File: persistent-storage-csi-azure
```

Comment:
Hi @lpettyjo,
I'm not sure why the section "Azure Disk CSI Driver Operator" will be removed. The "Azure Stack Hub" CSI Driver Operator (which is only installed on ASH) is not a replacement of the "Azure Disk" CSI Driver Operator on the Azure cloud.
From my understanding, "Azure Disk CSI Driver Operator" is still TP in 4.9, and "Azure Stack Hub CSI Driver Operator" is new and GA in 4.9. 

@tjungblu Please correct me if I missed something.

Reply:
Azure Disk CSI Driver Operator is the exact same as the Azure StackHub CSI Driver Operator - maybe adding the "Disk" would also help. 

CSI in ASH is GA, in Azure public cloud it is TP for 4.9.

Reply:
Yes, I think the CSI Driver are even the same one, but the operator deployment method are different which depends on the platform. 
I'm not sure if we need to merge as one section, but anyway, we need to keep both "Azure Disk CSI Driver Operator" and "Azure Stack Hub CSI Driver Operator", in one same section or two sections.

Reply:
My bad. Wasn't intentional. Thanks for catching this.